src/form_collection.py

```diff
@@ -0,0 +1,70 @@
+import os
+from flask import Flask, jsonify, request
+from pymongo import MongoClient
+
+
+
+access_url = os.environ['FORMS_EDITOR_SERVICE_DATASTORE_1_PORT_27017_TCP_ADDR']
```

Comment:
With the user defined network set up, this can be simplified to 
`access_url = '<service-name>:27017'`